docs/pages/blog/mui-x-end-v6-features.md

```diff
@@ -38,63 +38,64 @@ As we gear up to develop the next major version, let's review the notable new fe
 
 ### Charts
 
-The stable version of MUI X charts supports the most commonly used chart plots you'll need in your day-to-day applications.
+The stable version of MUI X Charts supports the most commonly used chart plots you'll need in your day-to-day applications.
+It features lines, areas, bars, pie charts, and scatter plots.
+And the latest version also include:
 
-Featuring lines, areas, bars, pie charts and scatter plots, the latest versions also include:
+#### Animations
 
-- **Animations**
-
-  Animations on bars, pie charts, and more to come.
+Bar and pie chars are the ones currently supporting it.
+More to come!
 
   <video preload="metadata" style="margin-bottom: 16px;" autoplay muted loop playsinline width="680" height="306">
     <source src="/static/blog/mui-x-end-v6-features/bar-chart-animations.mp4" type="video/mp4">
   </video>
 
-- **Horizontal Bar Chart Support**
+#### Horizontal bar Chart support
 
-  Recently added, the horizontal bars can broaden your data visualization spectrum, especially when you need to display long labels for multiple categories.
-  Check out Bar's [layout documentation](/x/react-charts/bars/#layout) for more details.
+Recently added, the horizontal bars can broaden your data visualization spectrum, especially when you need to display long labels for multiple categories.
+Check out [the Bar Chart layout documentation](/x/react-charts/bars/#layout) for more details.
 
-- **Improved Text Customization**
+#### Improved text customization
 
-  Customize the legend text to your heart's content, making your charts more readable and visually
-  appealing.
+Customize the legend text to your heart's content, making your charts more readable and visually
+appealing.
 
-[Get started with MUI X Charts now](/x/react-charts/getting-started/)!
+👉 [Get started with MUI X Charts now](/x/react-charts/getting-started/)!
 
 ### Tree view
 
-After an extensive period of development and refinement in the lab and pre-release packages, and with the invaluable feedback we've received from you, the community, we're happy to share that the package is now officially stable!
+After an extensive period of development and refinement in the lab and pre-release packages, and with the invaluable feedback we've received from you and the community, we're happy to share that the package is now officially stable!
```

Comment:
```suggestion
After an extensive period of development and refinement in the lab and pre-release packages, and with the invaluable feedback we've received from the community, we're happy to share that the package is now officially stable!
```

Or, we could also just say this! I know we were trying to convey that "you", as in the reader, is part of the community, but the sentence sounded weird to me. Let me know otherwise, happy to rollback to the original if I'm not making sense! 🤙